### ISSUE TITLE
Broken keyboard navigation within text color toolbar

### ISSUE THREAD

**Issue by ajlende** [WordPress/gutenberg#26115]
**Describe the bug**
Cannot navigate to hue selector or text input field within the custom color selector from the toolbar (see screenshot 2).

**To reproduce**
1. Type in some paragraph text
2. Open up custom color selector from toolbar (can be done with the keyboard)
3. Arrow keys correctly move the brightness/saturation selector
4. Tab does not navigate to the hue selector

**Expected behavior**
Pressing tab within the custom color selector should navigate to the hue selector (and then another tab should navigate to the text input, and so on).

**Screenshots**
<img width="560" alt="text color toolbar menu" src="https://user-images.githubusercontent.com/5129775/96006219-de770e80-0e02-11eb-8ce0-096a7f57b01b.png">
<img width="540" alt="text color toolbar's custom color popover" src="https://user-images.githubusercontent.com/5129775/96005880-86400c80-0e02-11eb-91f9-6e13c7c5d564.png">

**Editor version:**
- WordPress version: 5.5.1
- Does the website has Gutenberg plugin installed, or is it using the block editor that comes by default? Gutenberg plugin
- If the Gutenberg plugin is installed, which version is it? Master branch (b126b3f)

**Desktop:**
 - OS: MacOS
 - Browser: Chrome
 - Version: 85.0.4183.121

**Comment by ajlende** [comment]
Resolved by #27544
